src/super_gradients/common/registry/README.md

```diff
@@ -85,4 +85,45 @@ python main.py --config-name=my_recipe.yaml
 Coming soon
 
 ### C. Loss
-Coming soon
\ No newline at end of file
+
+*main.py*
+
+```python
+import omegaconf
+import hydra
+
+import torch
+from torch.nn.modules.loss import _Loss
+
+from super_gradients import Trainer, init_trainer
+from super_gradients.common.registry.registry import register_loss
+
+
+@register_loss("custom_rsquared_loss")
+class CustomRSquaredLoss(_Loss):
```

Comment:
add hare a comment that says we are extending torch.nn.modules.loss._Loss

Reply:
Something like that ? https://github.com/Deci-AI/super-gradients/pull/406/commits/5d2c13c6057900a73f55aa287fc4e68b16dcc7f5